asapdiscovery-data/asapdiscovery/data/dask_utils.py

```diff
@@ -0,0 +1,111 @@
+from enum import Enum
+from typing import Optional
+
+from dask import config as cfg
+from dask_jobqueue import LSFCluster
+
+try:
+    from pydantic.v1 import BaseModel, Field
+except ImportError:
+    from pydantic import BaseModel, Field
+
+from .execution_utils import guess_network_interface
+
+cfg.set({"distributed.scheduler.worker-ttl": None})
+cfg.set({"distributed.admin.tick.limit": "2h"})
+
+
+class GPU(str, Enum):
+    GTX1080TI = "GTX1080TI"
+
+    @classmethod
+    def get_values(cls):
+        return [gpu.value for gpu in cls]
+
+
+_LILAC_GPU_GROUPS = {
+    GPU.GTX1080TI: "lt-gpu",
+}
+
+_LILAC_GPU_EXTRAS = {
+    GPU.GTX1080TI: [
+        '-R "select[hname!=lt16]"'
+    ],  # random node that has something wrong with its GPU
+}
+
+
+def _walltime_to_h(walltime: str) -> int:
```

Comment:
use this as a validator, that way you can have the walltime in the correct format on the schema might be easier to track that way?

Reply:
fixed by removing mismatch between `24:00` and `24h` syntax altogether

Reply:
Just using `h` notation from now on.